app/views/placements/schools/mentors/show.html.erb

```diff
@@ -26,6 +26,10 @@
           <% row.with_value(text: @mentor.trn) %>
         <% end %>
       <% end %>
+
+      <%= govuk_link_to t(".remove_mentor"),
```

Comment:
Outside the scope of this PR, but if it's no trouble, worth doing here. The show page for school users shouldn't have the school name above the user name (line 12 above)

Reply:
Had a look at the page title, which is also not quite right. It should only be the user name, not the school name. I suppose the assumption is that school users don't need to be told which school they are using (since they only have one? Thought they might not have just one.... I suppose it will come out in testing, but in the meanwhile, it appears that is the assumption.

Reply:
Cool, it can be done in a separate bug ticket if it's too far out of scope.